CppSamples/DisplayInformation/ShowGrid/main.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright 2024 Esri.
```

Comment:
Yeah this looks like it's using an outdated template. They no longer look like this since the access token update, https://github.com/Esri/arcgis-maps-sdk-samples-qt/blob/v.next/Templates/CppSampleTemplate/main.cpp.tmpl